src/pages/DashboardPage.js

```diff
@@ -8,17 +8,38 @@ import Title from '../compontens/Title';
 import LatestBlogsPage from './LatestBlogsPage';
 import NewBlogPage from './NewBlogPage';
 import MyBlogsPage from './MyBlogsPage';
+import BlogPage from './BlogPage';
```

Comment:
If Blog is Page then it cannot be imported and used in Dashboard PAGE, it should be BlogComponent or FeedComponent to be more precise.

Reply:
Dashboard page is main page and it has Title and Sidebar components that is fixed, but NewBlogPage, LatestBlogsPage, MyBlogsPage and BlogPage are not fixed and they depend on route. Maybe better name for these components is Container, like BlogContainer, or like you said BlogComponent.

Reply:
Component is the right name